ydb/library/yql/providers/s3/proto/file_queue.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+option cc_enable_arenas = true;
+
+package NYql.NS3.FileQueue;
+
+import "ydb/library/yql/dq/actors/protos/dq_events.proto";
+import "ydb/public/api/protos/ydb_issue_message.proto";
+
+message TEvGetNextBatch {
+    optional NYql.NDqProto.TMessageTransportMeta TransportMeta = 100;
+}
+
+message TEvObjectPathBatch {
+    bool NoMoreFiles = 1;
+    repeated TObjectPath ObjectPaths = 2;
+
+    optional NYql.NDqProto.TMessageTransportMeta TransportMeta = 100;
+}
+
+message TEvObjectPathReadError {
```

Comment:
Do we need a separate event for it? Here we have the common event: https://a.yandex-team.ru/arcadia/contrib/ydb/library/yql/dq/actors/protos/dq_events.proto?rev=r13368411#L75

Reply:
I would prefer it to be separate because it was separate before and has special logic around it already

Reply:
Ok, let's leave it as it is